client/src/components/Register/MethodTile.scss

```diff
@@ -0,0 +1,64 @@
+.mfa-method-tile {
+  flex: 25%;
+  margin-right: $tile-spacing;
```

Comment:
```suggestion
```

By using `calc(33% - (#{$tile-spacing} / 2))` above and putting justify-content in the container you get this spacing done by flex (and no extra margin at the end of the row)